### PR TITLE
Explain editing of additional app charts

### LINKED ISSUE #1271: Document Editing of Helm chart(s)
**Is your feature request related to a problem? Please describe.**
For the new application helm charts, we should provide some mechanism to push updates or edit in place

**Describe the solution you'd like**
I'd ideally like to see a way to edit charts similar to `kubectl edit` or an editor in the UI

**Describe alternatives you've considered**
We could give access to the s3 bucket and let operators edit there but it seems dangerous

**Additional context**
This doesn't necessarily need to be done in the epinio cli but that's likely the easiest place to do this.

### PR DESCRIPTION
**Pull request by andreas-kupries**
Fix [#1271](https://github.com/epinio/epinio/issues/1271).

### CODE REVIEW

File: docs/howtos/create_custom_appcharts.md
Comment (nunix) [inline docs/howtos/create_custom_appcharts.md:95]:
```suggestion
> Note: we didn't change the chart version. Versioned
chart development is out of scope for this How-To.
```

File: docs/howtos/create_custom_appcharts.md
Comment (nunix) [inline docs/howtos/create_custom_appcharts.md:104]:
```suggestion
Given the plethora of possible options, this How-To simply assumes that the tarball is
available at the example URL:
```

File: docs/howtos/create_custom_appcharts.md
Comment (nunix) [inline docs/howtos/create_custom_appcharts.md:110]:
```suggestion
Copy the following text, change the `RELEASE_NAMESPACE` value to the namespace where Epinio was
installed into (by default it's `epinio`) and add it to a file of your choice:
```

File: docs/howtos/create_custom_appcharts.md
Comment (nunix) [inline docs/howtos/create_custom_appcharts.md:125]:
```suggestion
```

File: docs/howtos/create_custom_appcharts.md
Comment (nunix) [inline docs/howtos/create_custom_appcharts.md:144]:
```suggestion
You can also see the details of the chart in Epinio:
```

File: docs/howtos/create_custom_appcharts.md
Comment (nunix) [inline docs/howtos/create_custom_appcharts.md:150]:
```suggestion
## Troubleshooting
If the new Helm chart is not correctly registered in Epinio, check that the `RELEASE_NAMESPACE` was properly set in the CRD.
```

**Review by nunix — COMMENTED** [review]
Reviewed `create_custom_appcharts` only. Will have a second review for the `using_custom_appcharts`.

File: docs/howtos/create_custom_appcharts.md
Comment (andreas-kupries) [reply docs/howtos/create_custom_appcharts.md:43]: In the clone step one of the commands has the user setting `helm-charts/chart/application` as their working directory.
Thus the shorter relative path here is ok.

We could possibly rewrite the entire how-to to stay either outside the clone, or on the toplevel directory, and then use larger paths across the board. On the good side of that change the user would not have to track state in the form of the working directory.

File: docs/howtos/create_custom_appcharts.md
Comment (andreas-kupries) [reply docs/howtos/create_custom_appcharts.md:81]: See previous response regarding working directory in the executed commands.

File: docs/howtos/create_custom_appcharts.md
Comment (andreas-kupries) [reply docs/howtos/create_custom_appcharts.md:150]: `CRD` is the custom resource definition. `CR` is the custom resource and what we have here. Maybe better to expand the shorthand, or name it differently.

**Comment by andreas-kupries** [comment]
My plan is now to accept all these, and then re-tweak a few places I noted.

File: docs/howtos/using_custom_appcharts.md
Comment (nunix) [inline docs/howtos/using_custom_appcharts.md:90]:
```suggestion
You can also check which chart is set as `default` when listing the available charts.
As an example, the output below shows the chart `FOO` as the current default:
```

File: docs/howtos/using_custom_appcharts.md
Comment (nunix) [inline docs/howtos/using_custom_appcharts.md:95]:
```suggestion
| *       | FOO      | Foofy deployment           |
```

File: docs/howtos/using_custom_appcharts.md
Comment (nunix) [inline docs/howtos/using_custom_appcharts.md:102]:
```suggestion
You can set the system chart `standard` back as the default, by running the following command:
```

File: docs/howtos/using_custom_appcharts.md
Comment (nunix) [inline docs/howtos/using_custom_appcharts.md:109]:
```suggestion
:::note
The __empty string__ used in place of the chart name is mandatory. If you do not add it,
the command will only list the charts available.
:::
```

**Review by nunix — COMMENTED** [review]
Reviewed the `using_custom_appcharts` doc.
Rephrased to active voice and added admonitions.

File: docs/howtos/using_custom_appcharts.md
Comment (andreas-kupries) [reply docs/howtos/using_custom_appcharts.md:50]: Thanks, was not aware of this kind of syntax.

File: docs/howtos/using_custom_appcharts.md
Comment (andreas-kupries) [reply docs/howtos/using_custom_appcharts.md:31]: No. `standard` is the __system default__ and is automatically used when the user neither sets a local default, nor uses `--app-chart`. As such it is not set as the local default by default, and is not highlighted.

**Comment by andreas-kupries** [comment]
Thanks for the review. I accept all the changes and will then fix a few typos I saw in them.

**Comment by andreas-kupries** [comment]
Pushed my fixes to your fixes now.

**Review by nunix — APPROVED** [review]

**Pull request merged**
